src/main/java/cn/nukkit/block/BlockSlab.java

```diff
@@ -14,9 +14,6 @@
  */
 public abstract class BlockSlab extends BlockTransparentMeta {
 
-    public static final int SLAB_BLOCK_TYPE_BIT = 0x07;
```

Comment:
Why delete this?

Reply:
i thinks it's no longer working, i have change in the setDamage and in Level.java i have fixed :)

is it still needed?

Reply:
Obviously, there are a lot of places below that use 0x07 and 0x08. Using this variable instead of writing 0x07 directly is to better show the meaning of the parameter.

Reply:
oooh iam sorry :), wait repull :)